prime-router/src/testIntegration/kotlin/datatests/TranslationTests.kt

```diff
@@ -337,6 +351,14 @@ class TranslationTests {
             return hl7.encode().byteInputStream()
         }
 
+        private fun runSenderTransform(bundle: InputStream, schema: String): InputStream {
+            val fhirBundle = FhirTranscoder.decode(bundle.bufferedReader().readText())
+            val fhirConverter = FHIRConverter(metadata, settings)
+            val transformedBundle = fhirConverter.getTransformerFromSchema(schema)!!.transform(fhirBundle)
```

Comment:
Instead of creating a Fhir converter and calling `getTransofrmerFromSchema` you should be able to just call `FhirTransformer`'s constructor directly with `schema`. I guess it works either way though.

Reply:
Nice cleanup, thank you!